GitUI/CommandsDialogs/FormDeleteBranch.cs

```diff
@@ -104,7 +111,7 @@ private void Delete_Click(object sender, EventArgs e)
                 }
             }
 
-            GitDeleteBranchCmd cmd = new(selectedBranches, force: hasUnmergedBranches);
+            GitDeleteBranchCmd cmd = new(selectedBranches, force: true);
```

Comment:
This is the only place where we use `force` argument (outside the tests). Should we completely remove it?

Reply:
I would keep the argument `force` in order to emphasize it is added to the command.